torch/export/export_schema.py

```diff
@@ -0,0 +1,303 @@
+from dataclasses import dataclass
+from enum import auto, Enum
+from typing import List, Union, Dict
+
+
+################################################################################
+# Following section is the defining the permissible argument types for operators
+
+# Copied from torchgen/model.py
+class ScalarType(Enum):
```

Comment:
Hmm, should this be just an uniq identifier + bitsize? (I.e. why are we excluding quantized types?)

Reply:
we are not sure what to do with quantization yet. 
If export wish to support quantization, then we will need to add quantized types.